Makefile

```diff
@@ -219,7 +219,7 @@ kustomize-crd:
 # guidance https://github.com/kyverno/kyverno/wiki/Generate-a-Release
 release:
 	kustomize build ./definitions > ./definitions/install.yaml
-	kustomize build ./definitions > ./definitions/release/install.yaml
+	kustomize build ./definitions/release > ./definitions/release/install.yaml
```

Comment:
If we change to `./definitions/release`, won't Kustomize only look for resources under this particular folder? Then how do we sync all the changes to `./definitions/release/install.yaml`?

Reply:
> If we change to `./definitions/release`, won't Kustomize only look for resources under this particular folder? Then how do we sync all the changes to `./definitions/release/install.yaml`?

In release/kustomization.yaml we pull in `../bundle`

Reply:
Ops.. missed that! Thank you.